packages/@guidesmiths/cuckoojs-schematics/src/utils/package-json.utils.ts

```diff
@@ -0,0 +1,71 @@
+import {type Tree} from '@angular-devkit/schematics';
+import {
+	type ParseError,
+	type JSONPath,
+	type Segment,
+	parseTree,
+	printParseErrorCode,
+	modify,
+	applyEdits,
+} from 'jsonc-parser';
+
+export class PackageJsonUtils {
+	content: string;
+	host: Tree;
+	path: string;
+
+	constructor(host: Tree, path = './package.json') {
+		const buffer = host.read(path);
+		if (!buffer) {
+			throw new Error(`Path ${path} not found.`);
+		}
+
+		this.content = buffer.toString();
+
+		this.host = host;
+		this.path = path;
+
+		this.checkFileContent();
+	}
+
+	addPackage(name: string, version: string, dev = false) {
+		const section = dev ? 'devDependencies' : 'dependencies';
+		this.content = this.insertDependency([section, name], version, true);
+		this.host.overwrite(this.path, this.content);
+	}
```

Comment:
There is a util called  `addPackageJsonDependency` in `@nestjs/schematics/dist/utils/dependencies.utils` that perhaps we can use

Reply:
Yes, and this util is built based on that. However, I wanted to have something that could also manage scripts

Reply:
Cool!